Source/SwiftLintFramework/Rules/MissingDocsRule.swift

```diff
@@ -155,6 +155,10 @@ public struct MissingDocsRule: OptInRule {
     )
 
     public func validate(file: File) -> [StyleViolation] {
+        guard SwiftVersion.current == .two else {
+            warnMissingDocsRuleDisabledOnce
```

Comment:
Swift gives a `dispatch_once` fixit recommending to do this for Swift 3. It's what we do for SourceKit initialization in `Request.swift` too.